tang_test.go

```diff
@@ -127,13 +104,8 @@ func checkEncryptTang(t *testing.T, h crypto.Hash) {
 	if testing.Verbose() {
 		decryptCmd.Stderr = os.Stderr
 	}
-	if err := decryptCmd.Run(); err != nil {
-		t.Fatal(err)
-	}
-
-	if decryptedData.String() != inputText {
-		t.Fatalf("decryption decryption failed: expected '%s', got '%s'", inputText, string(encrypted))
-	}
+	require.NoError(t, decryptCmd.Run())
+	require.Equal(t, inputText, string(encrypted), "decryption failed")
```

Comment:
I believe it should be `string(encrypted)` -> `decryptedData.String()`

Reply:
Good catch!